src/output/help.rs

```diff
@@ -202,7 +201,7 @@ impl<'help, 'app, 'parser, 'writer> Help<'help, 'app, 'parser, 'writer> {
         debug!("Help::write_args");
         // The shortest an arg can legally be is 2 (i.e. '-x')
         let mut longest = 2;
-        let mut ord_m = VecMap::new();
+        let mut ord_m = BTreeMap::new();
```

Comment:
Side note: Since `VecMap` worked off of insertion order, we could potentially use `IndexMap`.  Unsure which is more optimal

Reply:
(ie we should use the more core type without real-world benchmarks justifying something more optimal, hence the "side not")